modules/vaos/app/models/vaos/appointment_time.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module VAOS
+  # This is a custom type class for ensuring Time is always coerced as UTC
+  class AppointmentTime < Virtus::Attribute
```

Comment:
We're moving away from Virtus since its deprecated. Maybe go ahead and use `dry-types` instead to avoid creating more debt?

Reply:
Yes, I've noticed that, but we've already used virtus all over the place and I don't have the time to grok dry-types right now. Will have to revisit as code debt unfortunately.

Reply:
This feature along with its in progress, sister PR, for cancelling an appointment request ideally should have been wrapped up by last week. I'm a little behind and will be out of pocket for the next couple days. I'm happy to open a ticket to make sure it's something we address along side swagger stuff that needs to be done too post launch.

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3546

Reply:
Swagger: https://github.com/department-of-veterans-affairs/va.gov-team/issues/3545

Reply:
I'm seeing

https://github.com/department-of-veterans-affairs/va.gov-team/issues/466
https://github.com/department-of-veterans-affairs/va.gov-team/issues/816

Is there why you feel there needs to be a greater sense of urgency for our team to address this? I certainly see value in doing making the switch for longer term maintainability, but I am having a hard time understanding the degree of urgency you're attributing here.